docs/get-started/running-nodes.mdx

```diff
@@ -4,306 +4,218 @@ title: 'Running Etherlink Nodes'
-    --apply-unsafe-patches
-```
-
-A few remarks.
-
-- `--apply-unsafe-patches` is only required the first time you start your node,
-  and can be removed from the command-line afterwards. Similarly, the
-  `unsafe-pvm-patches` field can be removed from the configuration field.
-- Once the Smart Rollup node has caught up, you can safely connect it to a
-  rolling `octez-node`.
-
-Once you have the rollup node running, you can confirm that you are able query
-it; for instance by requesting the latest read tezos block:
-
-```bash
-curl -s http://localhost:8932/global/block/head
-```
-
-### Running `octez-evm-node` in Observer Mode
-
-### From Genesis
-
-In order to run an `octez-evm-node` in observer mode from Genesis, you need:
-
-- An access to a Smart Rollup node tracking the state of Etherlink (see the
-  previous section to setup one yourself, which is probably necessary until
-  public nodes become available).
-- The `installer.hex` file that you can get by following the “Building
-  Etherlink’s Initial Kernel” appendix.
-
-```bash
-octez-evm-node init config --devmode \
-    --data-dir $evm_observer_dir --rollup-node-endpoint $sr_node_observer_rpc \
-    --preimages-endpoint https://snapshots.eu.tzinit.org/etherlink-mainnet/wasm_2_0_0 \
-    --evm-node-endpoint https://node.mainnet.etherlink.com \
-    --initial-kernel observer/installer.hex \
-    --force
-```
-
-If you do not want to relies on a public preimages endpoint (though it is worth
-mentioning again it is safe), you can move the `wasm_2_0_0/` directory built
-along the `installer.hex` file in the directory `$evm_observer_dir`. Note that
-this means you will have to provision this directory with the preimages of
-every kernel voted by the community and deployed on Etherlink after that.
-
-Starting the node is as simple as running this command.
-
-```bash
-octez-evm-node run observer --data-dir $evm_observer_node
-```
-
-By default, the `octez-evm-node` exposes its JSON RPC API endpoint to `localhost:8545`.
-You can test that everything works as expected by running some RPC requests manually or by
-setting your wallet to use your local node.
+## Running Etherlink nodes
+
+Etherlink nodes require access to the RPCs of a running Smart Rollup node.
+Therefore, to run an Etherlink node, you must also run a Smart Rollup node.
+You can set up your own layer 1 node or use a public node.
+
+The code for the three nodes is part of the [Octez suite](https://tezos.gitlab.io/).
+
+- You can use a public layer 1 node or set up your own as described in [Installing Octez](https://tezos.gitlab.io/introduction/howtoget.html) in the Octez documentation.
+- You can set up a Smart Rollup node (`octez-smart-rollup-node`) with a snapshot of the Etherlink state or from Etherlink genesis.
+Setting up the Smart Rollup node from Etherlink genesis requires a recent build of the Octez suite.
+- Octez does not yet provide a binary build of the Etherlink node, so you must build it from the source code.
```

Comment:
there is an docker images with it `tezos/tezos-bare`. it contains all binary even the experimental one.

it's an alternative to build it for now

Reply:
That's a bit tricky; the `latest` version of that image is 5 months old (7ec971fb193b) and does not have octez-evm-node.  People would have to use the specific tag docker pull tezos/tezos-bare:master_b9f6c913_20240424112144.

Reply:
How's this?

> Octez does not yet provide a binary build of the Etherlink node as part of its binary distribution or in the tezos/tezos docker image, but you can get it from the tezos/tezos-bare image if you use a specific tag. The tag master_b9f6c913_20240424112144 includes the Etherlink node and matches other examples on this page.